src/components/CollectionImage/CollectionImage.tsx

```diff
@@ -7,21 +8,25 @@ import { Props } from './CollectionImage.types'
 import './CollectionImage.css'
 
 export default class CollectionImage extends React.PureComponent<Props> {
+  static defaultProps = {
+    className: ''
+  }
+
   renderItemRow(items: Item[]) {
     return items.map((item, index) => <ItemImage key={index} item={item} />)
   }
 
   render() {
-    let { items, isLoading } = this.props
+    let { className, items, isLoading } = this.props
 
     const firstItemRow = items.slice(0, 2)
     const secondItemRow = items.slice(2, 4)
     const itemRowStyle = { height: secondItemRow.length ? '50%' : '100%' }
 
     return (
-      <div className="CollectionImage is-image">
+      <div className={classNames('CollectionImage is-image', className)}>
```

Comment:
Although it is the same, should we split them?
```suggestion
      <div className={classNames('CollectionImage', 'is-image', className)}>
```